### PR TITLE
CLOUDSTACK-8636: Verify failure of creation of Custom disk offering with disksize parameter

### PR DESCRIPTION
**Pull request by pritisarap12**
Custom disk creation should not take disksize parameter while creating the disk offering

### CODE REVIEW

**Comment by wilderrodrigues** [comment]
This description looks better, @pritisarap12 . I think you meant something like that for your previous PR.

I will wait for the build/travis and vote on them.

Cheers,
Wilder

File: test/integration/testpaths/testpath_custom_disk_offering.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    @attr(tags=["basic", "advanced"], required_hardware="false")
+    def test_create_custom_disk_offering_with_size(self):
+        """ Create custom disk offerign with size
+            1.   Create custom disk offering with size.
+            2.   Should not allow to create custom disk offering
+                 with size mentioned.(Exception should be raised)
+        """
+
+        with self.assertRaises(Exception):
```
Comment (sanju1010) [inline test/integration/testpaths/testpath_custom_disk_offering.py]: Due to product bug, if custom disk offering is created with size we need to delete it as part of cleanup. Can you please handle this?

Comment (pritisarap12) [inline test/integration/testpaths/testpath_custom_disk_offering.py]: Updated the testcase as per review changes.

**Comment by bhaisaab** [comment]
LGTM

**Comment by bhaisaab** [comment]
squashed/merged your commits; in future please try to squash them as you iterate with your reviews

**Pull request closed without merging**